jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/bsm/bsmDepositorToOde/BsmDepositorToOde.java

```diff
@@ -0,0 +1,82 @@
+package us.dot.its.jpo.ode.bsm.bsmDepositorToOde;
```

Comment:
Do we "have to"? These are in the test folder anyway. Ultimately, I think we need to turn these into more flexible and usable applications that can be used to read data from a file and send them to ODE.